src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,44 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp,liked, onLikeToggle}) => {
+
+  const handleHeartClick = () => {
+    onLikeToggle(id);
+  };
+
+  let isLocalUser = false
+
+  if (sender === 'Vladimir') {
+    isLocalUser = true;
+  } 
 
-const ChatEntry = () => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={isLocalUser?'chat-entry local':'chat-entry remote'}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p className='body'>{body}</p>
+        <TimeStamp time = {timeStamp}></TimeStamp>
```

Comment:
Nice job using the provided Timestamp component here. 

Convention is to not have spaces around `=` sign when passing props

```suggestion
        <TimeStamp time={timeStamp}></TimeStamp>
```